### ISSUE TITLE
Brave does not follow System Preference tab behavior

### ISSUE THREAD

**Issue by ballo** [brave/brave-browser#7531]
## Description 
Brave does not follow System Preference tab behavior
## Steps to Reproduce
Press tab
## Actual result:
tab will cycle through all UI elements
## Expected result:
tab should only cycle through text fields and maybe embedded lists (not popups). Tabbing through items in the Safari preferences can serve as an example.
## Reproduces how often: 
Every time.
## Brave version (brave://version info)
Version 1.1.23 Chromium: 79.0.3945.88 (Official Build) (64-bit)

## Miscellaneous Information:
I'll attach a screen shot of the relevant preference in its default setting
![Screen Shot 2019-12-28 at 1 04 27 AM](https://user-images.githubusercontent.com/631311/71540903-31ccae80-290e-11ea-81fb-8a32e5217941.png)

**Comment by bsclifton** [comment]
@ballo this is working for me as expected. As you shared above, keyboard navigation is not using tab by default. Safari does work anyways though. For Brave (or Chrome) you need to use Ctrl + Page Up / Ctrl + Page Down to cycle through tabs

If you enable the setting you shared, Brave DOES work with tab. Here's a screenshot (notice the orange border around the active tab):
<img width="1071" alt="Screen Shot 2020-01-03 at 2 37 49 PM" src="https://user-images.githubusercontent.com/4733304/71750999-5c9b9300-2e37-11ea-9761-f0529f1782f4.png">

It'll also give focus to the close button, which you can close using space
<img width="87" alt="Screen Shot 2020-01-03 at 2 38 00 PM" src="https://user-images.githubusercontent.com/4733304/71751017-6de49f80-2e37-11ea-9e23-200d70e81b5e.png">

Since everything is working as expected, I'm going to close this issue. If you are asking to always use tab (even if that setting is NOT checked), I don't think we're going to do that. Hope this is helpful 😄

**Comment by ballo** [comment]
@bsclifton it only works "as expected" if you enable the option I circled in red. If you do not, it does not work properly.

Please note that the option is disabled BY DEFAULT, and there are good reasons for that. This option was added for Windows users.

**Comment by ballo** [comment]
@bsclifton also you seem to be under the impression I was referring to window tabs, which I never was. This is referring to the TAB BUTTON i.e. the tab button on your keyboard.

**Comment by ballo** [comment]
@bsclifton please reopen this ticket as you obviously didn't understand the problem I reported.

**Comment by bsclifton** [comment]
@ballo I do believe I understand what you're talking about; it's not clear what you're asking for. With the setting you shared ENABLED, I'm able to cycle through the tabs just fine using TAB on the keyboard. The orange highlight being shown is from the OS showing that is the active tab. I can go backwards with SHIFT + TAB. 

By default, our tab behavior matches Chrome / Chromium. Pressing TAB will cycle through the elements on the active web page (this input box, the `Reopen and comment` button and `Comment` buttons that follow it, etc). Pressing SHIFT + TAB moves selection backwards

What is happening on your computer when you press TAB?

Safari does both of these. It will let you navigate through tabs, omnibox, and also through the active web page (even if the setting you shared is DISABLED)

**Comment by bsclifton** [comment]
![tab_behavior1](https://user-images.githubusercontent.com/4733304/71798060-dab19280-300d-11ea-8dd2-4c6aef608677.gif)

**Comment by ballo** [comment]
Here is a simple example. Keep the system-wide setting OFF.
https://www.tutorialspoint.com/html/html_forms.htm

**Comment by bsclifton** [comment]
@ballo is pressing tab not working for you? The GIF I have above demonstrates this perfectly. I have the setting OFF 
<img width="529" alt="Screen Shot 2020-01-06 at 9 27 30 AM" src="https://user-images.githubusercontent.com/4733304/71831752-c8177780-3066-11ea-89d3-678968dee8c0.png">

Can you please describe what is not working for you? and what happens instead?

**Comment by ballo** [comment]
In Safari it cycles through the text fields, following the global preference. In Brave it cycles though all the form controls.

**Comment by rebron** [comment]
Hi @ballo I see what you're requesting, however, we're not going to deviate from Chromium separately here and would need to wait on upstream fix from them if this is something they want to address which would then get incorporate later into Brave.

You can file an issue against Chromium here -> https://bugs.chromium.org/

**Comment by ballo** [comment]
LOL, they don't care about this platform. Chromium is just open source Chrome and google doesn't care about anything other than Chromebook and maybe Windows.

**Comment by rebron** [comment]
@ballo I know one of the accessibility guys on the Chromium team pretty well, been a good 15 years (?) making web browsers accessible. I forwarded him this issue. The care is there, how and when issues are addressed are always the challenge.

**Comment by ballo** [comment]
OK, sorry for being snippy but I've mentioned issues before and it's never taken as seriously as some obscure feature nobody uses and will soon die anyway like WebGL. Fingers crossed I guess
